src/components/tx/modals/NewTxModal/index.tsx

```diff
@@ -50,14 +59,21 @@ const NewTxModal = ({ onClose, recipient }: { onClose: () => void; recipient?: s
     setNftModalOpen(true)
   }
 
+  const onRejectModalOpen = () => {
+    trackEvent(MODALS_EVENTS.REJECT_TX)
+    setRejectModalOpen(true)
+  }
+
   const onContractInteraction = () => {
     trackEvent(MODALS_EVENTS.CONTRACT_INTERACTION)
     onClose()
   }
 
+  const dialogTitle = txNonce ? `Replace transaction #${txNonce}` : 'New transaction'
```

Comment:
Do we use the notation `#<nonce>` anywhere else? Or should we just write `Replace transaction with nonce <nonce>`

Reply:
We don't. I'll adjust this.